src/nwb2bids/_converters/_datalad_utils.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pathlib
+import platform
+
+
+def _read_first_bytes(file_path: pathlib.Path) -> bytes:
+    """
+    Read the first 6 bytes of a file; necessary for Windows 'symlinks'.
+    """
+    with file_path.open(mode="rb") as file_stream:
+        first_bytes = file_stream.read(6)
+        return first_bytes
```

Comment:
```suggestion
def _read_first_bytes(file_path: pathlib.Path, n: int = 6) -> bytes:
    """
    Read the first n (by default 6) bytes of a file; necessary for Windows 'symlinks'.
    """
    with file_path.open(mode="rb",) as file_stream:
        return file_stream.read(n)
```

to match the name better, to not require redoing when need comes, to make shorter since no point in defining variable just to return it